### DISCUSSION TITLE
GRASS GIS architecture documentation

### DISCUSSION THREAD

**Discussion by girishnand** [OSGeo/grass#3453] [General]
This, please. 

https://matklad.github.io/2021/02/06/ARCHITECTURE.md.html

**Comment by NikosAlexandris** [comment]
On 24/02/25 03:44πμ, girishnand wrote:
> This, please. 
> 
> https://matklad.github.io/2021/02/06/ARCHITECTURE.md.html

Great post!  Helps me further in a new project at my work.

Apologies for the noise.

Nikos

**Comment by neteler** [comment]
A basis might be:

https://grass.osgeo.org/programming8/
